.github/workflows/pull-request.yaml

```diff
@@ -0,0 +1,30 @@
+name: PR
```

Comment:
maybe rename this workflow pr-checklist.yaml
Most of the workflows in this directory are about doing stuff on pull-requests, so pull-request.yaml is not a very helpful name.

Reply:
Done.